demos/README.md

```diff
@@ -1,95 +1,68 @@
-
-6. Semi-Supervised Classification with Graph Convolutional Networks. T. Kipf, M. Welling.
-ICLR 2017. arXiv:1609.02907 ([link](https://arxiv.org/abs/1609.02907))
-
-7. Graph Attention Networks. P. Velickovic et al. ICLR 2018 ([link](https://arxiv.org/abs/1710.10903))
-
-8. On Calibration of Modern Neural Networks. C. Guo, G. Pleiss, Y. Sun, and K. Q. Weinberger.
-ICML 2017. ([link](https://geoffpleiss.com/nn_calibration))
-
-9. Predict then propagate: Graph neural networks meet personalized PageRank. J. Klicpera, A. Bojchevski, A., and S. Günnemann, ICLR, 2019, arXiv:1810.05997.([link](https://arxiv.org/abs/1810.05997))
-
-10. Cluster-GCN: An Efficient Algorithm for Training Deep and Large Graph Convolutional Networks. W. Chiang, X. Liu, S. Si, Y. Li, S. Bengio, and C. Hsiej, KDD, 2019, arXiv:1905.07953.([link](https://arxiv.org/abs/1905.07953))
-
-11. Axiomatic Attribution for Deep Networks. Mukund Sundararajan, Ankur Taly and Qiqi Yan. ICML 2017. ([link](https://arxiv.org/pdf/1703.01365.pdf))
+# StellarGraph demos
+
+
+[StellarGraph](https://github.com/stellargraph/stellargraph) provides numerous algorithms for graph machine learning. This folder contains demos of all of them to explain how they work and how to use them.
+
+The demo notebooks can be run without any installation of Python by using Binder or Google Colab - these both provide a cloud-based notebook environment.  The whole set of demos can be opened in Binder here: [![Open in Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stellargraph/stellargraph/master?urlpath=lab/tree/demos/README.md) or you can click the Binder and Colab badges within each notebook.
+
+## Find algorithms for a task
+
+- Predicting attributes, such as classifying as a class or label, or regressing to calculate a continuous number:
+  - For nodes/vertices/entities: [**node classification**](./node-classification)
+  - For edges/links/connections: [**link prediction**](./link-prediction) (includes knowledge graph completion)
+  - For graphs/networks: [**graph classification**](./graph-classification)
+  - Adjusting predictions scores to be probabilities (for any model): [**calibration**](./calibration)
+- Representation learning or computing embedding vectors (including unsupervised tasks):
+  - For nodes/vertices/entities and edges/links/connections: [**embeddings**](./embeddings) (and [**community detection**](./community_detection) for using these to cluster nodes)
+  - For graphs/networks: [**graph classification**](./graph-classification) (only supervised)
+- Ensembling models to reduce prediction variance: [**ensembles**](./ensembles)
+- Loading data into a `StellarGraph` object, with Pandas or NetworkX: [**basics**](./basics)
+- Experimental: running GraphSAGE on data stored in Neo4j: [**neo4j connector**](./connector/neo4j)
+
+## Find a demo for an algorithm
```

Comment:
This table does take a while to read, but I think it's a reasonable compromise in order to be able to make it fit vertically in markdown; I can't think of a better way.
Some possible suggestions:
* do we need to use all 'tick' glyphs here - some simpler combination of shapes and colours may be clearer?
* there's a mix in this and the other table of ✔️ and ✔ The smaller tick looks better but is too small; can we change the size enough with markdown to make it work?
* there's some small tick-box glyphs which don't look quite right, and aren't in the key below, eg:"☑ US"

Reply:
We've chatted a bit on slack, but in summary; I've switched to text, and it seems easier.

Reply:
Thanks, I find the new version much quicker to interpret; and as a bonus it doesn't need a key.